docs/expose-applications-using-loadbalancer-type-service.md

```diff
@@ -70,17 +70,52 @@ Request Body:
 
 ### Service annotations
 - loadbalancer.openstack.org/floating-network-id
+
+  The floating network id which will allocate floating ip for loadbalancer. This annotation works when the value of `service.beta.kubernetes.io/openstack-internal-load-balancer` is false.
 - loadbalancer.openstack.org/floating-subnet
+
+  A floating network can have several subnets. This annotation is one of subnets' name.
 - loadbalancer.openstack.org/floating-subnet-id
+
+  This annotation is one of subnets' id in a floating network.
 - loadbalancer.openstack.org/class
+
+  The name which predefined floating networks and subnets in config file.
 - loadbalancer.openstack.org/subnet-id
+
+  The subnet id which loadbalancer vip will be created on it.
 - loadbalancer.openstack.org/port-id
+
+  The vip port id for loadbalancer.
 - loadbalancer.openstack.org/connection-limit
+
+  The maximum number of connections per second allowed for the listener. Positive integer or -1 for unlimited (default).
 - loadbalancer.openstack.org/keep-floatingip
+
+  If 'true', the floating ip will be deleted together with loadbalancer. Default is false.
```

Comment:
If 'true', the floating ip will **NOT** be deleted

Reply:
done